src/db/tables/featureFlags.ts

```diff
@@ -44,7 +44,8 @@ export type FeatureFlagName =
   | "MonthlyActivityEmail"
   | "CsatSurvey"
   | "CancellationFlow"
-  | "ConfirmCancellation";
+  | "ConfirmCancellation"
+  | "FirstDataBrokerRemovalFixedEmail";
```

Comment:
Depending on whether we intend to make this a real experiment (waiting on confirmation) we should consider using the deprecated feature flags. This would spare us from experiment checks for each individual user.

Reply:
@rhelmer and I discussed with PM that for the emails it might be better to just use the feature flag here. We don’t intend to run an experiment and having just an on/off switch is handy for the cronjob.

Reply:
> Depending on whether we intend to make this a real experiment (waiting on confirmation) we should consider using the deprecated feature flags. This would spare us from experiment checks for each individual user.

Reply:
We might want to re-consider deprecating our own flag system since it's proved to be useful... I think doing slow roll-out is a good general practice so I'm still a bit on the fence about it.

Reply:
However in this case I agree let's use Monitor's own feature flag system for this feature.